app/new_service_manifest.yml

```diff
@@ -18,7 +18,17 @@
   name: Service type
   editable: true
   questions:
-    - serviceTypes
+    - serviceTypesIaaS
+-
+  name: Service type
```

Comment:
Any reason these sections need to be repeated? Could we have one section with the three questions in?